packages/build/src/tarball.ts

```diff
@@ -111,12 +150,19 @@ export async function createTarball(
       path: filename,
       contentType: 'application/gzip'
     };
+  } else if (buildVariant === BuildVariant.Redhat) {
+    await tarballRedhat(input, outputDir, version, rootDir);
+
+    return {
+      path: filename,
+      contentType: 'application/x-rpm'
+    }
   } else if (buildVariant === BuildVariant.Debian) {
     await tarballDebian(input, outputDir, version, rootDir);
 
     return {
       path: filename,
-      // this might have to be application/gtarball MIME type
+      // this might have to be application/gzip MIME type
       contentType: 'application/vnd.debian.binary-package'
```

Comment:
I’m not sure how this is used, but fwiw, this is the official mime type for `.deb`: https://www.iana.org/assignments/media-types/application/vnd.debian.binary-package

Reply:
we use this to upload the artifact to evergreen's S3 bucket. I will be leaving it with the `content-type` I have, which is also the one you found, i.e. `application/vnd.debian.binary-package`. (removed that comment cause it's confusing at this point)